hera_cal/tests/test_cal_formats.py

```diff
@@ -25,6 +25,8 @@ def test_gainHC(self):
             elif param == '_extra_keywords':
                 continue
             else:
+                if "_antenna_" in param:
+                    param = param[1:]
```

Comment:
What is the purpose of this statement? It's not clear.

Reply:
that is because for whatever reason, a `getattr(hc, '_antenna_numbers') == `getattr(uv, '_antenna_numbers')` fails but a similar comparison of `getattr(hc, 'antenna_numbers') == `getattr(uv, 'antenna_numbers')` does not fail.. i'll add a comment